desktop/backend/python/controller/desktop.py

```
@@ -0,0 +1,173 @@
+        with self._lock:
+            return list(self.mouse_history)
+
+    # =================================================
+    # Window information
+    # =================================================
+
+    def get_open_windows(self) -> List[str]:
+        try:
+            windows = gw.getAllWindows()
+            return [w.title for w in windows if w.title]
+        except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)